scripts/scenario_03_waves.lua

```diff
@@ -3,8 +3,10 @@
 ---
 --- Spawn the player ships you want. The strength of enemy ships is independent of the number and type of player ships.
 -- Type: Basic
--- Variation[Hard]: Difficulty starts at wave 5 and increases by 1.5 after the players defeat each wave. Players are overwhelmed more quickly, leading to shorter games.
```

Comment:
If variations are replaced with settings, I would expect the call to `getScenarioVariation` to be replaced as well.